frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -347,12 +374,19 @@ export const removeResourceConnection = (
     return Promise.reject();
   }
 
-  const replaceValue =
-    _.get(target, ['metadata', 'labels', 'app.kubernetes.io/instance']) || target.metadata.name;
+  const replaceTargetName =
+    target.metadata?.labels?.['app.kubernetes.io/instance'] ?? target.metadata?.name;
```

Comment:
@sbose78 should we continue to reference the instance name as part of the `connects-to` label value now that it is an object? Or still use the resource name. IMO it makes more sense now to use the resource name.

Reply:
@sbose78 ^^

Reply:
You mean, here, right?
```
{ "apiVersion": "apps/v1", "kind": "Deployment", "name": "Foobar" }
```
This got to be the resource name.